From 7d8932980f2ebfdd39f0c3495bae9eaf3230678b Mon Sep 17 00:00:00 2001
From: Yan Markman <ymarkman@marvell.com>
Date: Tue, 20 Mar 2018 12:05:00 +0200
Subject: [PATCH 0574/1051] Documentation/bindings: marvell: add 7k-8k board
 descriptions

Adding descriptions for Marvell Armada 7K/8K boards

Change-Id: I70cac7979b6a7fef2050084b0b75152602120dcb
Signed-off-by: Yan Markman <ymarkman@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 .../bindings/arm/marvell/armada-7k-8k.txt           | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/Documentation/devicetree/bindings/arm/marvell/armada-7k-8k.txt b/Documentation/devicetree/bindings/arm/marvell/armada-7k-8k.txt
index df98a9c82a8c..1daadaccf8e2 100644
--- a/Documentation/devicetree/bindings/arm/marvell/armada-7k-8k.txt
+++ b/Documentation/devicetree/bindings/arm/marvell/armada-7k-8k.txt
@@ -18,6 +18,19 @@ the following root node property:
    - "marvell,armada8040", "marvell,armada-ap806-quad", "marvell,armada-ap806"
       when the SoC being used is the Armada 8040
 
+Boards:
+   - "marvell,armada7040-db-default", "marvell,armada7040-db"
+      when the board being used is the Armada 7040 Development board
+
+   - "marvell,armada8040-db-default", "marvell,armada8040-db"
+      when the board being used is the Armada 8040 Development board
+
+   - "marvell,armada8040-mcbin"
+      when the board being used is the MACCHIATOBin community board
+
+   - "marvell,armada8040-mcbin-single-shot"
+      when the board being used is the MACCHIATOBin single shot community board
+
 Example:
 
 compatible = "marvell,armada7040-db", "marvell,armada7040",
-- 
2.17.1

